content/en/real_user_monitoring/mobile_and_tv_monitoring/unity/advanced_configuration.md

```diff
@@ -1,15 +1,13 @@
 ---
-title: RUM Unity Advanced Configuration
+title: Unity Advanced Configuration
```

Comment:
I think Advanced Configuration still needs to be added under Unity in the nav, see [the staging page](https://docs-staging.datadoghq.com/rtrieu/rum-mobile-sdk-docs-restructure/real_user_monitoring/mobile_and_tv_monitoring/unity/advanced_configuration/)